src/main/java/step5/domain/Cars.java

```diff
@@ -0,0 +1,43 @@
+package step5.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+    List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public Cars() {
+
+    }
```

Comment:
기본 생성자를 제거하고 차량 이름들을 받는 파라미터를 받는 생성자를 추가하여
RacingGame의 createCars() 메서드 기능을 해당 생성자로 옮겨서 수행한다면 어떤 장점이 있을까요 ~ ?
```suggestion
    public Cars(final String names) {
        ...
    }
```